source/ncengine/graphics2/frontend/subsystem/WireframeRendererSubsystem.cpp

```diff
@@ -0,0 +1,151 @@
+#include "WireframeRendererSubsystem.h"
```

Comment:
This whole file is almost copy paste from the existing implementation. Changes renderer type to `MeshRenderer2` and slightly tweaked a few `emplace_back` calls to match `WireframeInstanceState`.

Reply:
If possible, I think we should not have any editor-specific logic in the engine itself, and we should leave it up to the user whether or not to use the component.